src/components/IdentityOptions/IdentityOptions.tsx

```diff
@@ -17,9 +18,12 @@ interface Props {
 export function IdentityOptions({ identity, onEdit }: Props): JSX.Element {
   const t = browser.i18n.getMessage;
 
-  const { address } = identity;
+  const { address, did } = identity;
 
-  const { buttonProps, itemProps, isOpen, setIsOpen } = useDropdownMenu(3);
+  const isLightDid = Boolean(did) && !isFullDid(did);
```

Comment:
Should it also check whether the DID has been deleted?

Reply:
Isn't that already checked by `syncDidStateWithBlockchain` which sets did to undefined if it was deleted?

Reply:
Sorry, I misread the code, all good :+1: